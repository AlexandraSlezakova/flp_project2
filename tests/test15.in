A B
A C
C D
C E
D E
E F
D F
