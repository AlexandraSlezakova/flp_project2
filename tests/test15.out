A-B A-C C-D C-E D-F
A-B A-C C-D C-E E-F
A-B A-C C-D D-E D-F
A-B A-C C-D D-E E-F
A-B A-C C-D D-F E-F
A-B A-C C-E D-E D-F
A-B A-C C-E D-E E-F
A-B A-C C-E D-F E-F
